config.json

```diff
@@ -357,7 +357,6 @@
       },
       "settings": {
         "homepage_featured_products_count": 6,
-        "homepage_featured_products_column_count": 3,
```

Comment:
I am not sure I understand how does this break click ability. As a developer I should be able to override the setting in each variation over the default. It should not break the theme.

Reply:
@junedkazi you're right, the underlying issue was introduced with #948.  i've reverted that change and this one fixes both issues.